task3-4/task3-4/checkScreen/CheckViewController.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  CheckViewController.swift
+//  task3-4
+//
+//  Created by Григоренко Александр Игоревич on 23.09.2022.
+//
+
+import UIKit
+
+/// VC сцена с чеком
+class CheckViewController: UIViewController {
+    
+    @IBOutlet weak var progressView: UIProgressView!
+    private var completeAlert: UIAlertController = UIAlertController()
+    private var progressBarTimer: Timer!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.progressBarTimer = Timer.scheduledTimer(timeInterval: 0.1, target: self,
```

Comment:
```suggestion
         self.progressBarTimer = Timer.scheduledTimer(timeInterval: 0.1, target: self,
                                                     selector: #selector(updateProgressView), userInfo: nil,
                                                     repeats: true)
        completeAlert = UIAlertController(title: " Ваш заказ готов! ", message: nil, preferredStyle: .alert)
        completeAlert.addAction(UIAlertAction(title: " Хорошо! ", style: .cancel))
```
вынести в метод из viewDidLoad